recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>you can say here that inference_mark_lte is used to filter assets that have an agreement score lower than 51%</p><p> </p><p>you could use 50% directly, since lte stands for "lower than or equal to"</p><p> </p><p>you can say that 671 examples have been wrongly classified, that is the model was only right 16% of the time</p><p> </p><p>we have 8 classes, so a random "dumb" model would be able to have 1/8 = 12.5% accuracy.</p><p> </p><p>So our LLM is a little bit smarter than a dumb model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I've made this change. Closing</p>